pkg/config/config.go

```diff
@@ -118,10 +118,8 @@ func (idCfg *IdentityConfig) loadFromENV() error {
 
 	// parse values
 	var err error
-	idCfg.PodIP = net.ParseIP(idCfg.rawPodIP)
-	if idCfg.PodIP == nil {
-		// PodIP should always be non-nil to issue role certificate
-		return fmt.Errorf("Invalid POD_IP [%q]", idCfg.rawPodIP)
+	if idCfg.rawPodIP != "" {
+		idCfg.PodIP = net.ParseIP(idCfg.rawPodIP)
```

Comment:
PodIPのparseに失敗した場合は、errorが発生する仕様になってると思いますので、以下のような追記をしたほうがよいかなと思いましたがいかがでしょうか。
```suggestion
		idCfg.PodIP = net.ParseIP(idCfg.rawPodIP)
		if idCfg.PodIP == nil {
			return fmt.Errorf("Invalid POD_IP [%q], %w", idCfg.rawPodIP, err)
		}
```